app/views/dashboard/_node_default.html.erb

```diff
@@ -4,9 +4,9 @@
     <%= render partial: 'dashboard/node_moderate', locals: { node: node } %>
 
     <% if node.main_image %>
-        <a class="img" href="<%= node.path %>"><img src="<%= node.main_image.path(:default) %>" style="width:100%;" /></a>
+        <a class="img" href="<%= node.path %>"><%= image_tag(node.main_image.path(:default), style:'width:100%;') %></a>
```

Comment:
Hi @Tlazypanda - this seems to have disrupted the loading of some images which are "blob" type - see for example the admittedly odd ones in this view:

https://publiclab.org/embed/grid/grid:infragram-upload

See how these actually use a dataurl as the image? I know its weird but I see the `image_tag` method is replacing the filename with `blob` - can we include a workaround, perhaps, if the image path is a data url? 

This is a pretty weird workflow... i feel like maybe I'm missing something!

Also, I'm trying to fix the somewhat related assets on that template (for use in embedded content), here:
https://github.com/publiclab/plots2/commit/36c54a21a3fc1b94c8e8c0981b09e76cb1e9a1af

Reply:
Hey @jywarren I was thinking of writing a conditional rendering for this but am having some difficulty in defining the condition I was thinking of something like:- 
`<% elsif current_user.photo.url.start_with?("data:image")  %>` but doesn't seem to work... My understanding of the models may not be at point here can you help me out :sweat_smile:

Reply:
Yeah, hmm. Are we mixing up blobs with data_urls? This is a bit mysterious! Uh, is it possible that the lazy load gem is converting a data-url into a blob? 

AH! I remember, i wrote some really obscure code here. Let's see...

OMG YES -- https://github.com/publiclab/plots2/pull/6931 and https://github.com/publiclab/plots2/pull/6930 -- deep mysterious code! But yes, it has the data-uri-to-blob method there. That should help us!

Reply:
Should we consider just turning off lazy loading for this one instance in order to recover the functionality? @TildaDares @Manasa2850 @17sushmita are any of you interested in this particular bug? Thank you all!

Reply:
I was looking into this and was playing around with some modifications on the unstable but I'm not sure how to reproduce this on my local. Would love to get any help regarding this.

Reply:
Unfortunately I'm not sure how to reproduce... I'm not sure how the blob images are saved! Can we dig into the raw html shown on unstable to try to learn more?

Reply:
My best guess here is that we should just turn off lazy loading for this one type of image.

Reply:
OK this file is for the old dashboard! So ignore this particular file, we'll revert the one specifically for the new card-style display of nodes, below, i'll leave a comment.